test/lshr_shl_i1_regularize.ll

```diff
@@ -0,0 +1,57 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv -s %t.bc -o %t.reg.bc
```

Comment:
I think this testcase is identical for lshr and shl.  If so, consider using the pre-processor to refactor the test:

; RUN: %clang %s -DOPCODE=shl -E > %t.shl.ll
; RUN: llvm-as %t.shl.ll -o %t.bc
; [rest of run commands]

; RUN: %clang %s -DOPCODE=lshr -E > %t.lshr.ll
; RUN: llvm-as %t.lshr.ll -o %t.bc
; [rest of run commands]

Reply:
IMHO as is it's more readable

Reply:
Agree with @MrSidims that it's better to keep the tests written as is.